be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/Resources/AccountResources.en.resx

```diff
@@ -207,6 +216,9 @@
   <data name="SignInText" xml:space="preserve">
     <value>Sign in</value>
   </data>
+  <data name="SmsMarketingApprovalLabel" xml:space="preserve">
+    <value>I consent to telephone contact from ELTAP for marketing purposes</value>
```

Comment:
**suggestion:** Word ELTAP couldn't be in text, because this app is not used only by ELTAP